portal/src/util/formatDatetime.ts

```diff
@@ -18,5 +17,5 @@ export function formatDatetime(
   }
 
   datetime = datetime.setLocale(locale);
-  return datetime.toLocaleString(format);
+  return datetime.toFormat("ff ('UTC'ZZ)");
```

Comment:
Per offline discussion, we will still use toLocaleString, but with a custom format options.

Please include this link above the option object as comment for future reference. https://tc39.es/ecma402/#sec-datetimeformat-abstracts

Reply:
updated, thanks